src/main/java/ca/team2706/frc/robot/subsystems/DriveBase.java

```diff
@@ -368,6 +429,28 @@ public void setPositionNoGyro(double speed, double setpoint) {
         follow();
     }
 
+    /**
+     * Goes to a position with the closed loop Talon PIDs using only encoder and gyro
+     *
+     * @param speed          The speed from 0 to 1
+     * @param setpoint       The setpoint to go to in feet
+     * @param targetRotation The desired rotation
+     */
+    public void setPositionGyro(double speed, double setpoint, double targetRotation) {
```

Comment:
Not sure if this would actually help at all, but if in theory the speed, setpoint and the targeRotation were the same after the method was called multiple times, wouldn't this lead to the possibility of spamming the talons with commands? 

@eandr127 maybe you'll be able to tell whether or not this is a thing we should consider.

Reply:
I think it's fine because it's pretty much the same thing as sending a motor speed every tick like we would do with arcade drive. If it isn't called, the motor safety watchdog might disable the motor.